tf_quant_finance/black_scholes/vanilla_prices.py

```diff
@@ -293,9 +293,9 @@ def barrier_price(*,
     the barriers option under black scholes.
 
   """
-  if (continuous_dividends is None) == (cost_of_carries is None):
-    raise ValueError('At most one of continuous_dividends and cost of carries '
-                     'may be supplied')
+  if (continuous_dividends is not None) == (cost_of_carries is not None):
```

Comment:
actually continuous dividends and cost_of_carries can both be None and should be treated like [here](https://github.com/google/tf-quant-finance/blob/848762e3da280601acdb571c2b6e971aa6670418/tf_quant_finance/black_scholes/vanilla_prices.py#L150)